main.go

```
@@ -65,12 +65,15 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	watchNamepace := os.Getenv("WATCH_NAMESPACE")
```

Can you panic if this is not set?

---

done